src/Utopia/Messaging/Adapters/Push/Pushy.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Push;
+
+use Utopia\Messaging\Adapters\Push as PushAdapter;
+use Utopia\Messaging\Messages\Push;
+
+class Pushy extends PushAdapter
+{
+    /**
+     * @param  string  $secretKey The secret API  key that pushy provides.
+     */
+    public function __construct(
+        private string $secretKey,
+    ) {
+    }
+
+    /**
+     * Get adapter name.
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'Pushy';
+    }
+
+    /**
+     * Get max messages per request.
+     *
+     * @return int
+     */
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1000;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
+    public function process(Push $message): string
+    {
+
+        return $this->request(
```

Comment:
This should build a structured response consistent with other adapters, please check APNS/FCM for an idea on how to do so